src/bundle/ui-dev/src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -260,6 +260,7 @@ const UniversalDiscoveryModule = (props) => {
         return {
             isInitLocationsDeselectionBlocked: props.isInitLocationsDeselectionBlocked,
             initSelectedLocationsIds: props.selectedLocations,
+            initSelectedItems: props.initSelectedItems,
```

Comment:
nitpick - I guess it's `initiallySelectedItems` , `init` sounds for me a little as function that's supposed to initialize selected items (whatever that means)

Reply:
I wanted to be consistent with the previous naming convention - `initSelectedLocationsIds `.